metrics/prometheus/prometheus.go

```diff
@@ -0,0 +1,302 @@
+package metrics
+
+import (
+	"github.com/prebid/prebid-cache/config"
+	"github.com/prometheus/client_golang/prometheus"
+	"time"
+)
+
+// Constants and global variables
```

Comment:
Please remove the "header" comments in this file:

- Constants and global variables
- Object definition
- Init functions
- Helper Init functions
- `CacheMetrics` interface implementation

Reply:
done